wmigroup/copy_files.txt

```diff
@@ -0,0 +1 @@
+wmigroup.resource.ps1
```

Comment:
I'm not sure about auto-enabling this WMI provider in default DSCv3 deployment.
Discovery in this provider takes ~5-10 seconds and returns ~1300 resource types. This is a significant perf hit especially when a user is not interested in WMI resources. Related issue #274 .

Reply:
This one, at least for now, we might want to document opting-in until #274 is handled. Otherwise it'll give the impression that DSC is _very_ slow and require users not interested in WMI during the alpha to opt out. I also expect that to impact test runs.

Reply:
For now, I'll rename `wmigroup.dsc.resource.json` to `wmigroup.dsc.resource.json.optout` so that the `WMIGroup` resource is still copied to the release folder but will be skipped by default.